platform/darwin/src/NSExpression+MGLAdditions.h

```diff
@@ -173,6 +173,8 @@ FOUNDATION_EXTERN MGL_EXPORT const MGLExpressionInterpolationMode MGLExpressionI
  guide for a correspondence of operators and types between the style
  specification and the `NSExpression` representation used by this SDK.
  
+ Note that the `length` argument is not currently supported with iOS 8.
```

Comment:
Since this is important for devs to know it may be better if this is changed to `@note`:

>@note The <a href="https://www.mapbox.com/mapbox-gl-js/style-spec/#expressions-length">length </a> attribute from the style spec is not supported in iOS8.